src/style.css

```
@@ -0,0 +1,66 @@
+body {
+  margin: auto;
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 2px;
+  margin-left: 2px;
+  background-position: center;
+  background-repeat: no-repeat;
+  flex-shrink: 0;
+}
+
+.stars__star:last-child {
+  margin-right: 2px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--0 .stars__star:nth-child(-n+5) {
```

```suggestion
.stars--0 .stars__star:nth-child(-n+5),
.stars--1 .stars__star:nth-child(n+2),
... {
```
Consider using commas to group selectors and avoid repeating code